build/x86_64_debian/Dockerfile

```diff
@@ -1,33 +1,10 @@
-FROM debian:7
-
-RUN \
- # Point to the archive...
-    sed -i \
-      -e 's/\(deb\|httpredir\)\.debian\.org/archive.debian.org/g' \
-      -e '/-updates main/d' \
-      -e '/http\.debian\.net/d' \
-      /etc/apt/sources.list \
- # ...and disable the expiration check...
- && echo 'Acquire::Check-Valid-Until "0";' \
-      >/etc/apt/apt.conf.d/10no-check-valid-until \
- # ...and remove pointers to the defunct security repos.
- && sed -i \
-      -e '/security\.debian\.org/d' \
-      /etc/apt/sources.list
+FROM debian:9
 
 RUN apt-get -y update \
- # Needed for autoconf, due to broken Debian 7 repos.
- && apt-get -y -o Apt::Get::force-yes=1 install perl-base=5.14.2-21+deb7u3 \
- # Needed for g++, due to broken Debian 7 repos.
- && apt-get download libc-bin=2.13-38+deb7u10 \
- && dpkg -i libc-bin_2.13-38+deb7u10_amd64.deb \
- && apt-get download libc6=2.13-38+deb7u10 \
- && dpkg -i libc6_2.13-38+deb7u10_amd64.deb \
- && apt-get -y install linux-libc-dev \
- # Needed for rvm, due to broken Debian 7 repos.
- && apt-get -y install gnupg2 procps \
- && apt-get download libssl1.0.0=1.0.1e-2+deb7u20 \
- && dpkg -i libssl1.0.0_1.0.1e-2+deb7u20_amd64.deb \
+ # Needed for the gpg2 command below.
+ && apt-get -y install linux-libc-dev gnupg2 \
+ # Needed for rvm.
+ && apt-get -y install procps \
```

Comment:
Why install these separately, and not add them to the list below?

Reply:
I kept these two separate because they are dependencies of rvm and gpg2 instead of the agent build itself.

Reply:
See followup in https://github.com/GoogleCloudPlatform/google-fluentd/pull/254.